### DISCUSSION TITLE
Garage Door Opener Settings Problem

### DISCUSSION THREAD

**Discussion by Sepphuber007** [mongoose-os-apps/shelly-homekit#1548] [General]
Hello!

Is it possible to add deactivate the closing sensor as a 3 point in the garage door opener with Shelly 1 Plus and Home Kit software because my door does not have a light barrier or sensor and I always get blocked when closing. I use a Normstahl door with Magic 1000-2 motor.

Many thanks for your help in advance and 

best regards 

Sepp

**Comment by markirb** [comment]
Currently no. And I think it also makes no sense or you can tell me how the Shelly should stay in sync with the state of the door if you reboot or something else happens.

Just get and wire a small reed sensor for ~10€ and you will be happy 😊 if the Shelly is on 230V you also need the addon.

**Reply by Sepphuber007** [reply]
I don't know where to attach the sensor, unfortunately how it works.

The door works perfectly this way.

- When opening it comes open
- When closing it comes out blocked although it is closed.

Do you happen to know a solution how I can do it?

How do you do that?  Do you attach the sensor to the shelly or gate?
Just get a small tongue sensor for ~10€ and wire it and you are happy 😊 if the Shelly is connected to 230V you also need the addon. 

Thank you very much 

Translated with DeepL.com (free version)

(PS: Wir können auch in deutsch schreiben wenn sie wollen )

**Comment by Sepphuber007** [comment]
![s-l1200](https://github.com/user-attachments/assets/1e32ffa4-56bc-46ef-8ea3-1096bc6e823b)

**Reply by Sepphuber007** [reply]
![grafik](https://github.com/user-attachments/assets/bac319ba-5347-468b-acf5-b4540b563402)

**Comment by markirb** [comment]
You wire it to Shelly

see here for diagram

https://github.com/mongoose-os-apps/shelly-homekit/wiki/Garage-Door-Opener-Settings

**Comment by Sepphuber007** [comment]
Kann man den Shelly 1 Plus( Homekit) auch mit den Shelly Plus Addon betreiben bzw muss ich den Shelly Plus Addon auch auf Homekit flashen ? Wie funktioniert das 

Bild: ![grafik](https://github.com/user-attachments/assets/7e8ee924-8de0-4e54-8edf-ea23575ff828)






lg Sepp

**Reply by markirb** [reply]
Nothing to flash. Just Plug & Play. All Plus Models are supported.

**Comment by Sepphuber007** [comment]
ok Den Shelly 2.5 gibts es fast nicht mehr geht das auch mit dem nachfolger Shelly Plus 2PM ?

**Reply by markirb** [reply]
Warum nen neuen Shelly?

Das wiring von +1 geht wie bei 1. 

Ich würde schauen das der Shelly auf DC läuft. Irgendwo in dem garagencontroller kann man die meist abgreifen. 
Dann braucht man nur den kontaktsensor und fertig.

Ansonsten auf AC laufend auf den +1 das addon mit kontaktschalter und auch fertig.
